src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/entrypoint/Entrypoint.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable import/no-named-as-default */
+import { AltinnAppHeader, AltinnContentIconFormData, AltinnContentLoader, AltinnModal } from 'altinn-shared/components';
+import { AltinnAppTheme } from 'altinn-shared/theme';
+import { IParty } from 'altinn-shared/types';
+import * as React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import ProcessStep from 'src/shared/containers/ProcessStep';
+import { startInitialStatelessQueue } from 'src/shared/resources/queue/queueSlice';
+import { IRuntimeState, ProcessTaskType } from 'src/types';
+import Form from '../form/containers/Form';
+import Instantiate from '../instantiate/containers';
+
+export default function Entrypoint() {
+  const profile = useSelector((state: IRuntimeState) => state.profile.profile);
+  const selectedParty = useSelector((state: IRuntimeState) => state.party.selectedParty);
+  const applicationMetadata = useSelector((state: IRuntimeState) => state.applicationMetadata.applicationMetadata);
+  const [action, setAction] = React.useState<string>(null);
+  const statelessLoading: boolean = useSelector((state: IRuntimeState) => state.isLoading.stateless);
+  const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    if (applicationMetadata) {
+      const onEntry = applicationMetadata.onEntry;
+      if (!onEntry || onEntry.show === 'Instantiate') {
```

Comment:
What is the difference between `new-instance` and `Instantiate`? Should `Instantiate` be part of the interface?
```
export interface IOnEntry {
  show: 'new-instance' | 'startpage' | string;
}
```

Reply:
Just me being lazy. `new-instance` should be used :)